recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -15,14 +13,20 @@ RDEPENDS_${PN}-host-tools = "aktualizr aktualizr-repo aktualizr-cert-provider ${
 PV = "1.0+git${SRCPV}"
 PR = "7"
 
+GARAGE_SIGN_PV = "0.5.0-4-gdd4744f"
```

Comment:
Why pick a version from five months ago and not the latest? Just for testing?

Reply:
sorry, that's what was sitting in one of my build directories... I'll fix